client/app/components/Input/Input.scss

```diff
@@ -147,11 +147,10 @@
 
   &Label {
     @include setMargin($size-0, $size-0, $size-0, $size-8);
-
+    color: $white;
```

Comment:
Is this necessary?

Reply:
yes, otherwise checkbox label turns brown in color.

Reply:
Double-checked and that looks fine! Thanks!